packages/cli/src/commands/upgrade.js

```diff
@@ -115,18 +116,46 @@ export const handler = async ({ dryRun, tag, verbose, dedupe }) => {
         title: 'One more thing..',
         task: (ctx, task) => {
           const version = ctx.versionToUpgradeTo
-          task.title =
+          const messageSections = [
             `One more thing...\n\n   ${c.warning(
               `🎉 Your project has been upgraded to RedwoodJS ${version}!`
-            )} \n\n` +
+            )} \n\n`,
             `   Please review the release notes for any manual steps: \n   ❖ ${terminalLink(
               `Redwood community discussion`,
               `https://community.redwoodjs.com/search?q=${version}%23announcements`
             )}\n   ❖ ${terminalLink(
               `GitHub Release notes`,
               `https://github.com/redwoodjs/redwood/releases` // intentionally not linking to specific version
-            )}
-          `
+            )} \n\n`,
```

Comment:
How about skipping all of this if a tag is specified? (Except maybe for RCs)

Reply:
Sorry skip what section? The additional messages trigger if you specify any tag and then the two conditions inside are:
1. Am I on a tag I don't currently watch for updates?
2. Am I on the canary tag?

Reply:
GH is being buggy 🙁 That comment was ment to be attached to lines 123-129. Basically remove the links to the community forums announcement search and the GH release notes, because neither of those will exist for canary

Reply:
Ah okay. What tags are these links valid for? In principle someone can run `upgrade -t latest` to switch from canary to latest - in that case they have a tag and we'd still want to show?

Valid for "latest" and "rc"?

Reply:
Good point about `latest`. So yeah, `latest` and `rc` sounds good.
But I wonder how useful that link is anyway. Try this link for example https://community.redwoodjs.com/search?q=4.0.0%23announcements

At least for me, this is the result 😆
![image](https://user-images.githubusercontent.com/30793/216073481-a1c3d46d-468e-468b-9b3d-fe71f031860f.png)

Reply:
Haha good point! I mean if you search for `1.0.0` you also get all the `1.0.0-rc.6` and if you search for `2.0.0` you get a link to a post about Prisma `2.0.0` in redwood `0.10.0`!

Maybe these links need a bit of an update then. Probably need to discuss what to change them to though?

Reply:
Yeah, we should discuss. And that can go into another PR. So for now, just to get rid of some of the duplication/information overload, I suggest just hiding them unless upgrading to latest or RC.

But let's also wait for @jtoar's input

Reply:
Even for RCs that search is not very helpful. It'll search for something like "4.1.0-rc.230", and that's not how we name RC topics. It's going to be something more generic, like "v4.1 RC" or something like that.

Reply:
> I suggest just hiding them unless upgrading to latest or RC

This sounds good